convert2rhel/utils.py

```diff
@@ -66,6 +69,141 @@ class Color(object):
 BACKUP_DIR = os.path.join(TMP_DIR, "backup")
 
 
+# Code taken from https://stackoverflow.com/a/63773140
+class Process(multiprocessing.Process):
+    """Overrides the implementation of the multiprocessing.Process class."""
+
+    def __init__(self, *args, **kwargs):
+        """Default constructor for the class"""
+        multiprocessing.Process.__init__(self, *args, **kwargs)
+        self._pconn, self._cconn = multiprocessing.Pipe()
+        self._exception = None
+
+    def run(self):
+        """Overrides the `run` method to catch exceptions raised in child."""
+        try:
+            multiprocessing.Process.run(self)
+            self._cconn.send(None)
+        # Here, `SystemExit` is inherit from `BaseException`, which is too
+        # broad to catch as it involves for-loop exceptions. The ideal here
```

Comment:
```suggestion
        # broad to catch as it involves for-loop exceptions. The idea here
```
Unsure if you meant ideal or idea

Reply:
It's `idea`. Thanks!